fl4health/clients/fenda_client.py

```diff
@@ -120,6 +125,14 @@ def get_cosine_similarity_loss(self, local_features: torch.Tensor, global_featur
         assert len(local_features) == len(global_features)
         return torch.abs(self.cos_sim(local_features, global_features)).mean()
 
+    def get_mkmmd_loss(self, local_features: torch.Tensor, global_features: torch.Tensor) -> torch.Tensor:
+        """
```

Comment:
This docstring is probably a copy-paste oversight, as it discusses cosine similarity?